.env.example

```diff
@@ -1,2 +1,15 @@
 PORT=''
-SECRET=''
\ No newline at end of file
+SECRET=''
+CLIENTID_PROD = ''
+CLIENTSECRET_PROD = ''
+GOOGLE_CALLBACK_URL_PROD = ''
+CLIENTID_TEST = ''
```

Comment:
@sebaztyn we should the remove the spaces in-between so we don't confuse the users. `CLIENTID_TEST = ''` should be like `CLIENTID_TEST=''`

Reply:
I made made the corrections are advised